src/clojure/clojurewerkz/spyglass/client.clj

```diff
@@ -56,6 +56,11 @@
       (.setProtocol cfb ConnectionFactoryBuilder$Protocol/BINARY))
     (.build cfb)))
 
+(defn- cas-value->map
+  "Given a CASValue, transform it to a map with keys #{:cas :value}"
+  [^CASValue cas-value]
+    {:value (when cas-value (.getValue cas-value))
```

Comment:
I think we can drop the `when`s. If `.getValue` and friends return `null`, the result is going to be the same. And I assume that's the idea here?